client/src/components/Playground/Playground.js

```diff
@@ -339,6 +342,12 @@ const Playground = ({history, typingDone, setTypingDone}) => (
         <div className="footer-text">
           Backround svg - Free Vector Graphics by <a href="https://www.vecteezy.com">Vecteezy!</a>
         </div>
+        <div className="footer-text">
+          Icons made by <a href="https://www.freepik.com/" title="Freepik">Freepik</a> from
```

Comment:
Now I've noticed one thing - there are missing spaces between texts and links. But I really don't know why:
![image](https://user-images.githubusercontent.com/38027246/54813342-99721100-4c8d-11e9-932b-dbf7e31f8e38.png)